validator/example/cmd.go

```diff
@@ -17,21 +17,29 @@ func NewCommand() *cobra.Command {
 	}
 
 	cmd1 := &cobra.Command{
-		Use:     "subcmd",
+		Use:     "subcmd01",
```

Comment:
Do you mind to use for loops for creating commands.
It will look better without DRY